test/tests/styleTest.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+describe("Zotero.Styles", function() {
+	var styleID = "http://www.zotero.org/styles/cell";
+	var styleFile = getTestDataDirectory();
+	var style;
+	styleFile.append('cell.csl');
```

Comment:
Better to avoid `nsIFile` when it's easy to do so:

`var styleFile = OS.Path.join(getTestDataDirectory().path, 'cell.csl');`

(Then at some point we'll fix `getTestDataDirectory()` to return a string path and can just take out the `.path`.)